react-hot-loader/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-hot-loader
+// Project: https://github.com/gaearon/react-hot-loader
+// Definitions by: Jacek Jagiello <https://github.com/jacekjagiello/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Actually, this definition is made for version 3(3.0.0-beta.6), I will add version number.